Automattic/vip-go-mu-plugins
# Add WP-CLI command to manage option autoload

## Description

Add command for managing option autoload. See #453

Also https://github.com/trepmal/option-autoload-cli

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change has relevant documentation additions / updates (if applicable). **has inline docs**

## Steps to Test

With PR applied, can test with:

```
wp option autoload get home # should be 'yes' at start
wp option autoload set home no
wp option autoload get home # should be 'no'
wp option autoload set home yes # restore
wp option autoload list no # should list starting with `moderation_keys`, `recently_edited`
```
